download-ipfs-binary.bash

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+set -e
```

Comment:
Please, re add `set -x`: it will allow us to debug this process as we run this script from CI pipelines